docs/core-concepts/run-udfs/run.mdx

```diff
@@ -208,18 +207,27 @@ fused.run(my_udf, engine="realtime")
 
 Set `sync=False` to run a UDF [asynchronously](/core-concepts/async/).
 
-### Passing arguments in `fused.run()`
+### Passing arguments in [`fused.run()`](/python-sdk/top-level-functions/#run)
 
-{/* 
-Need to write how to pass arguments 
-- when to pass arguments (bbox most common, then other args)
-- how to do it
-- linking to batch process?
-*/}
+A typical [`fused.run()`](/python-sdk/top-level-functions/#run) call of a UDF looks like this:
 
-🚧 Under construction
+```python showLineNumbers
+@fused.udf
+def my_udf(inputs: str):
+    import pandas as pd
+    return pd.DataFrame({"output": [inputs]})
+
+fused.run(my_udf, inputs="hello world")
+```
+
+A [`fused.run()`](/python-sdk/top-level-functions/#run) call will require the following arguments:
+1. [Mandatory] The first argument needs ot be the UDF to run (name, object, token, etc [as seen above](/core-concepts/run-udfs/run-small-udfs/#fusedrun))
+2. [Optional] Any arguments of the UDF itself (if it has any). In the example above that's `inputs` because `my_udf` takes `inputs` as arguments.
```

Comment:
```suggestion
2. [Optional] Any arguments of the UDF itself (if it has any). In the example above that's `inputs` because `my_udf` takes `inputs` as argument.
```